lition/README.md

```diff
@@ -2,13 +2,25 @@
 
 ## Price API
 
+Returns the price in Euros per MWh
+
 ### Endpoint
 
 https://staking.lition.io/api/v1/energy/source/1/date/2020-10-07/hour/15/
 
+### Sources
+
+| Number | Source                |
+| ------ | --------------------- |
+| 1      | agora-energiewende.de |
+| 2      | smard.de              |
+| 3      | energy-charts.info    |
+
 ### Input Params
 
 - `source`: The provider to retrieve price data from (required, e.g. "1", "2", or "3")
+- `date`: The date to query formatted by `[YEAR][MONTH][DAY]` e.g. `20201012`, if not provided defaults to the current UTC date (optional)
```

Comment:
The reason it is formatted this way is because in the recent COVID adapter it took in a date parameter in this format. 

Not sure if there is already a convention for how we want to normalize date params, but I adopted it from there. This is a completely separate adapter use case so I could see the argument for formatting it how the API wants.